cmd/orchestrator.go

```diff
@@ -62,8 +66,38 @@ var cmdOrchestrator = &cobra.Command{
 			app.Logger.Fatal(err)
 		}
 
+		fleetDBClient, err := fleetdb.NewFleetDBClient(ctx, app.Config, app.Config.ConditionDefinitions, app.Logger)
+		if err != nil {
+			app.Logger.Fatal(err)
+		}
+
+		// init Orchestrator API server
+		optionsOrcAPI := []server.Option{
+			server.WithLogger(app.Logger),
+			server.WithListenAddress(app.Config.ListenAddress),
+			server.WithStore(repository),
+			server.WithFleetDBClient(fleetDBClient),
+			server.WithStreamBroker(streamBroker, app.Config.NatsOptions.PublisherSubjectPrefix),
+			server.WithConditionDefinitions(app.Config.ConditionDefinitions),
+			server.WithAsOrchestrator(facility),
```

Comment:
The name of the option here throws me: we're already running as an Orchestrator, and you're only specifying the facility here as an option. Why change the name of this?

Reply:
I'll change `WithAsOrchestrator` to `WithOrchestratorAPI` in a follow-up PR. 
I do not follow what you mean by 'Why change the name of this?' - since this is code being added.

For sake of a common convention going ahead - we currently have a `Conditions` API - which is consumed by clients requesting conditions. And going ahead, these changes introduce an `Orchestrator` API which is for consumption by controllers within our 'control plane'.